src/Sentry.Analyzers/Sentry.Analyzers.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <IsPackable>false</IsPackable>
+        <Nullable>enable</Nullable>
+        <LangVersion>latest</LangVersion>
+
+        <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+        <IsRoslynComponent>true</IsRoslynComponent>
+
+        <RootNamespace>Sentry.Analyzers</RootNamespace>
+        <AssemblyName>Sentry.Analyzers</AssemblyName>
```

Comment:
afaik these default to the project name so are redundant 
```suggestion
```

Reply:
I removed them